### ISSUE TITLE
Publish to Chrome store

### ISSUE THREAD

**Issue by rodolfomiranda** [WebOfTrust/signify-browser-extension#138]

**Comment by edeykholt** [comment]
Back in December, I posted Blocktrust.dev's Identity Wallet browser extension for PRISM into the Chrome Web Store.

Here are some tips for getting through the publishing process of Polaris to Chrome Web Store.   This is from memory, but from what I recall:
* need a company behind the publishing process, essentially
* will require a Privacy Policy and commitment to some Google Chrome privacy policies
* need an email for customer support
* need a physical mailing address
* if manifest v3 permissions aren't minimal, there will be challenges or delays from Google.  Try to move any permissions into optional permissions and prompt the user during the user experience. Granted permissions will be remembered by Chrome.  The manifest v3 permisison names aren't always obviously indicative of the required user experience, since the permissions work together.  For example, the manifest should indicate the user will be prompted whether they want the extension to have permission to the current tab.  For the blocktrust browser extension, I added some code to support this.   Attached are what I  ended up publishing with back in the fall (other than version changes).  Note that no host_permissions are defined; rather, optional_host_permissions are defined. 

[manifest.json](https://github.com/WebOfTrust/signify-browser-extension/files/14791303/manifest.json)
